### DISCUSSION TITLE
dall-e-3 Unable to fetch the picture in the conversation

### DISCUSSION THREAD

**Discussion by igeekbb** [open-webui/open-webui#1086] [Q&A]
As shown, it's constantly loading... Unable to fetch the image.
![CleanShot 2024-03-07 at 20 38 19@2x](https://github.com/open-webui/open-webui/assets/154116268/321a7020-1542-4a72-96c1-63bda25f1654)
Tried drawing twice but couldn't get the picture
![CleanShot 2024-03-07 at 20 40 22@2x](https://github.com/open-webui/open-webui/assets/154116268/586fc868-2a4e-4415-86b6-431648957184)

**Comment by justinh-rahb** [comment]
DALL-E image generation APIs are not supported currently.

**Reply by igeekbb** [reply]
是否有计划支持DALL-E API？

**Comment by reiebrole30** [comment]
![image](https://github.com/open-webui/open-webui/assets/139945395/3a96eda2-8981-4b2e-82e5-08076304cde7)
mine works just fine

**Reply by justinh-rahb** [reply]
You're not doing the same thing the previous posters were trying.
